edx-platform/pearson-theme/lms/templates/courseware/course_about_sidebar_header.html

```diff
@@ -7,62 +7,124 @@
 from django.urls import reverse
 from django.conf import settings
 from six import text_type
+from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
+%>
+<%
+social_share_itp = configuration_helpers.get_value('social_share_itp', ' ')
```

Comment:
Since this setting is a boolean value, the default should be False instead of ' '. @amalbas